connection_pool/connection_pool.go

```diff
@@ -0,0 +1,732 @@
+// Package with methods support master discovery feature
```

Comment:
Maybe something like "Package with methods to work with a Tarantool cluster considering master discovery."?

Reply:
Updated.